### PR TITLE
03_As a user, I need to modify my announcement

### PR DESCRIPTION
**Pull request by Jean-Beru**
Issue #9

### CODE REVIEW

File: src/SensioLabs/JobBoardBundle/Resources/views/forms.html.twig [hunk 1]
```diff
@@ -59,7 +59,13 @@
             <div class="padding clearfix">
                 <button class="green-button right" name="submit" type="submit">
                     <span class="sprite-icon-light-arrow-right"></span>
-                    <span>Preview</span>
+                    <span>
+                        {% if fromPreview is defined and fromPreview %}
```
Comment (iamluc) [inline src/SensioLabs/JobBoardBundle/Resources/views/forms.html.twig]: shorter `{% if fromPreview|default(false) %}`

Comment (ogizanagi) [inline src/SensioLabs/JobBoardBundle/Resources/views/forms.html.twig]: Could even be inlined as a ternary operator then

File: src/SensioLabs/JobBoardBundle/Controller/JobController.php [hunk 2]
```diff
@@ -31,7 +32,7 @@ public function showAction(Job $job)
      *
      * @param Request $request
      *
-     * @return array
+     * @return array|RedirectResponse
```
Comment (romqin) [inline src/SensioLabs/JobBoardBundle/Controller/JobController.php]: I'm not fan of PHPDoc like `@param` or `@return` in controllers.
There is type hinting and we know that a controller must return a response.
Plus `@Route`,`@ParamConverter`,`@Template` are less visible.

**Comment by salahm** [comment]
:+1:

**Comment by romqin** [comment]
:+1:

**Pull request merged**